supplier_app/forms.py

```diff
@@ -93,4 +154,5 @@ class Meta:
             'bank_name': forms.TextInput(attrs={'class': 'form-control'}),
             'bank_code': forms.TextInput(attrs={'class': 'form-control'}),
             'account_number': forms.TextInput(attrs={'class': 'form-control'}),
```

Comment:
Should we be consistent with names? all information related to `bank_account` should be appended with `bank_account_` or something similar?

Reply:
Ok i will consider that